docs/editors-guide/rare-disease-curator.md

```diff
@@ -0,0 +1,51 @@
+# Rare Disease Curation
+
+Rare diseases in Mondo are created based on 3 main subsets. Rules for editing these subsets depends on the subset type. Before editing rare disease information in Mondo, please familiarize yourself with how rare diseases are represented in Mondo [here](https://mondo.readthedocs.io/en/latest/editors-guide/rare-disease-subset/).
+
+It is important to remember that the rare disease subsets are updated automatically at the time of the monthly Mondo release. It is, therefore, crucial that editing rare disease information is done by following the instructions below for these edits to be retained after the release.
+
+**1. Rare disease from authority sources.**
+These terms have an `in_subset` annotation `X_rare`  (where X is the rare disease authority).
+These subsets are maintained automatically:
+- a list of rare diseases is maintained for each source here:
+    - src/ontology/subsets/nord-subset.template.tsv
+    - src/ontology/subsets/gard-subset.template.tsv
+- during the Mondo release, the `X_rare` subset is updated based on the Mondo terms on the authoritative source list
+
+Until we have set a process with the sources to maintain this list automatically, Mondo editors should update this list of rare diseases manually **upon request by the authoritative source**.
+1. Create or open a GitHub issue reported by the authoritative source (ie NORD, GARD) to add or remove a term from the authoritative rare disease list.   
+1. Open the rare disease list (from src/ontology/subsets) in a text editor:
+     1. To add a new term, add a new line with the Mondo ID to add to the rare subset. Please make sure that the line is similar to the previous one: this document is a robot template, and therefore the information in each column is important.
+     1. To remove a term from the list, delete the line
+1. The term will be added to the rare disease list once the PR is merged, but the subset annotation will not be added until the monthly Mondo release has run.
+
+Note: rare diseases from Orphanet are maintained by a completely automated process. If an update is needed, please create a GitHub issue for the technical team.
```

Comment:
@matentzn is that right?

Reply:
The subset is automatically generated yes. But the mappings to Orphanet are of course curated.